x-pack/test/functional/apps/graph/feature_controls/graph_security.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+import { SecurityService } from 'x-pack/test/common/services';
+import { KibanaFunctionalTestDefaultProviders } from '../../../../types/providers';
+
+// tslint:disable:no-default-export
+export default function({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) {
+  const esArchiver = getService('esArchiver');
+  const security: SecurityService = getService('security');
+  const PageObjects = getPageObjects(['common', 'graph', 'security', 'error']);
+  const testSubjects = getService('testSubjects');
+  const appsMenu = getService('appsMenu');
+
+  describe('security', () => {
+    before(async () => {
+      esArchiver.load('empty_kibana');
+      // ensure we're logged out so we can login as the appropriate users
+      await PageObjects.security.logout();
+    });
+
+    after(async () => {
+      // logout, so the other tests don't accidentally run as the custom users we're testing below
```

Comment:
should we unload `empty_kibana` here?

Reply:
Maybe? We're rather inconsistent with the state which we leave everything in after these feature control tests... I'm torn on what it means to be an ideal "test neighbor"... I've found that some other tests expect that Kibana is found in a "default state", including the ability to logout a user... so  when we unload the empty_kibana es archive, it requires that subsequent tests load some esArchive which restores the .kibana index to a usable state.

Reply:
I thought empty_kibana might be a special case... I'm fine leaving this as-is